test/system/liquidation.test.js

```diff
@@ -131,18 +144,33 @@ describeFn("System -- liquidation", () => {
       // Deposit has been liquidated.
       expect(await tbtcDeposit1.currentState()).to.equal(11) // LIQUIDATED
 
-      // Signer bonds should land on the risk manager contract.
+      // The percentage of signer bonds that should be sent to the risk manager
+      // contract consists of the initial 66% and a portion of the remaining 34%
+      // that depends on the time passed before take offer. The percentage
+      // (rounded to the whole number) is therefore equal to:
+      // (66 + 34 * ((25920 + 22870)/86400)) = 85
       await expect(tx).to.changeEtherBalance(
         riskManagerV1,
-        bondedAmount.mul(bondedAmountPercentage).div(100)
+        bondedAmount.mul(85).div(100)
+      )
+    })
+
+    it("should transfer collateral tokens to the bidder", async () => {
+      expect(await collateralToken.balanceOf(bidder.address)).to.be.closeTo(
+        to1e18(6), // 30% of the initial asset pool
+        precision
+      )
+      expect(await collateralToken.balanceOf(assetPool.address)).to.be.closeTo(
```

Comment:
How about extracting this assertion to a separate `it`? Ex. it("should adjust asset pool's collateral tokens after the claim")

Reply:
I extracted it to a separate `it`, as suggested.

Reply:
I extracted it to a separate it, as suggested.